src/framework/theme/components/actions/actions.component.ts

```diff
@@ -93,6 +93,21 @@ export class NbActionComponent {
   protected _disabled: boolean = false;
   static ngAcceptInputType_disabled: NbBooleanInput;
 
+  /**
+   * Use badge dot mode
+   * @type boolean
+   */
+  @Input()
+  @HostBinding('class.badge-dot')
```

Comment:
badge would have dot-mode class anyway, why do we need to add additional binding here?